Userland/Libraries/LibWeb/MimeSniff/Resource.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "MimeType.h"
```

Comment:
The include style is typically the full path:
```suggestion
#include <LibWeb/MimeSniff/MimeType.h>
```